src/v2/Artsy/Router/RouterLink.tsx

```diff
@@ -71,7 +71,12 @@ export const RouterLink: React.FC<RouterLinkProps> = ({
         href={to as string}
         className={(props as LinkPropsSimple).className}
         style={Object.assign(styleProps, (props as LinkPropsSimple).style)}
-        {...omit(props, ["activeClassName", "hasLighterTextColor"])}
+        {...omit(props, [
+          "activeClassName",
+          "alignItems",
+          "hasLighterTextColor",
+          "underlineBehavior",
+        ])}
```

Comment:
`alignItems` isn't supported in either Link or a?

It just seems like there should be an actual typeguard then omit the small number of props that are specific to the found link.